int32 car_id

int64 traj_id
time start_time
float64 extra_yaw

geometry_msgs/Point[] pos_pts
geometry_msgs/Vector3 init_v
geometry_msgs/Vector3 init_a
float64[] t_pts
